myapp/spec/system/maintenance_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Maintenance' do
+  let(:file_path) { 'tmp/maintenance.txt' }
+
+  after do
+    FileUtils.rm_f(file_path)
+  end
+
+  describe 'start maintenance' do
+    it 'show maintenance page', :aggregate_failures do
+      Rake::Task['maintenance_go'].invoke
```

Comment:
[FYI]
It might be better to cut ’rake task’ out as a subject.
```ruby
escribe 'start maintenance' do
    subject { Rake::Task['maintenance_go'].invoke }

    it 'show maintenance page', :aggregate_failures do
      subject

      expect(File).to exist(file_path)
      visit '/'
      expect(page).to have_content 'Maintenance time! (503)'
    end
```